src/app/components/common/navigation/navigation.component.ts

```diff
@@ -33,6 +33,9 @@ export class NavigationComponent extends ViewControllerComponent implements OnIn
     this.iconTypeMenuTitle = this.navService.iconTypeMenuTitle;
     // Loads menu items from NavigationService
     this.navService.menuItems$.subscribe(menuItem => {
+      this.ws.call("failover.licensed", []).subscribe((res) => {
+            _.find(_.find(menuItem, { state: "system" }).sub, {state: 'failover'}).disabled = !res;
+          });
```

Comment:
It will be great if we can start linting at least any new code we add.

Reply:
Improved formatting